src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultArray = {};
+  const splitString = sourceString.split(';');
+
+  splitString.map((element) => {
+    if (element.length >= 2) {
+      const splitElement = element.split(':').map((item) => item.trim());
+
+      resultArray[splitElement[0]] = splitElement[1];
+    }
+  });
```

Comment:
The condition 'element.length >= 2' might not be sufficient to ensure that the element is a valid CSS declaration. It would be better to check if the element contains a colon, which is more indicative of a CSS property-value pair.